src/javascript/crypto/e2e/otr/session.js

```diff
@@ -279,4 +279,13 @@ e2e.otr.Session.prototype.getPublicKey = function() {
 e2e.otr.Session.prototype.getSigner = function() {
   return this.context_.getSigner();
 };
+
+
+/**
+ * Initiates OTR AKE, sending DH_COMMIT and entering AUTHSTATE_AWAITING_DHKEY.
+ */
+e2e.otr.Session.prototype.initiateOtr = function() {
+    this.send(new e2e.otr.message.DhCommit(session));
```

Comment:
Yeah session should be this.  I thought I patched it (https://github.com/google/end-to-end/commit/de4162264f1b45bc8939728d57c5928a4d0bf8b7) already but seems not to have made it here